### PR TITLE
fix: distance_range calculation

### PR DESCRIPTION
**Pull request by maqi**
### Summary generated by Reviewpad on 09 Aug 23 11:24 UTC
This pull request makes a minor correction to the `.github/workflows/merge.yml` file. It changes the name of a test from "execute the dbc spend test" to "execute the dbc spend tests".

### CODE REVIEW

File: sn_node/src/replication.rs [hunk 1]
```diff
@@ -57,10 +57,12 @@ impl Node {
         }
 
         // set the distance range used for pruning
-        let distance_range = match our_close_group.get(CLOSE_GROUP_SIZE) {
+        // our_close_group now only have CLOSE_GROUP_SIZE entries,
+        // which makes `our_close_group.get(CLOSE_GROUP_SIZE)` return None always.
```
Comment (joshuef) [inline sn_node/src/replication.rs:87]: I'm not sure I follow why this is always None?

Comment (maqi) [reply sn_node/src/replication.rs:87]: `our_close_group` now only contains 8th entries, 
but `.get(CLOSE_GROUP_SIZE)` is asking for the entry with `index 8`.
As the vec's indexing is from 0, hence it's actually looking from `9th entry`, which is `None`

Comment (joshuef) [reply sn_node/src/replication.rs:87]: Ah, i'd perhaps remove this line ```// which makes `our_close_group.get(CLOSE_GROUP_SIZE)` return None always.``` as it's a bit confusing (as we dont do it?)

We can just not perhaps ```// our_close_group now only have CLOSE_GROUP_SIZE entries, so we grab the last entry```

maybe `last and farthest` if they are sorted?

Comment (maqi) [reply sn_node/src/replication.rs:87]: Ah, that comment is mainly explain what's the issue of previous, i.e. `our_close_group.get(CLOSE_GROUP_SIZE)` .
Not about the current approach (using `last()`).

**Comment by joshuef** [comment]
already in main

**Pull request closed without merging**